### DISCUSSION TITLE
Many Missing Modules

### DISCUSSION THREAD

**Discussion by Meilings95** [cnr-isti-vclab/TagLab#53] [Q&A]
Hello,

I'm new to python so bear with me...

I ran the install.py and it was successful but when I try to open taglab I'm constantly getting warnings that I don't have the correct modules installed. Is there another way/step I missed? I've been going through one at a time manually installing the missing modules but there is no end in site and I feel like there has to be a better way. TIA

**Comment by maxcorsini** [comment]
Hello,

TagLab works despite this warnings ? Please, post them so that we can give you a suitable answer.
